x/tx/go.mod

```diff
@@ -35,6 +37,8 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
+replace cosmossdk.io/api => ../../api // TODO remove after api module is tagged
```

Comment:
I purposely put below that we shouldn't have replace api 😅

Reply:
Im Fine blocking this pr till the tag of API since its not blocking any other work